base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -615,6 +615,58 @@ impl ConsensusConstants {
         vec![consensus_constants_1]
     }
 
+    /// *
+    /// Stagenet has the following characteristics:
+    /// * 2 min blocks on average (5 min SHA-3, 3 min MM)
+    /// * 21 billion tXTR with a 3-year half-life
+    /// * 800 T tail emission (± 1% inflation after initial 21 billion has been mined)
+    /// * Coinbase lock height - 12 hours = 360 blocks
+    pub fn stagenet() -> Vec<Self> {
+        // Note these values are all placeholders for final values
+        let mut algos = HashMap::new();
+        algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
+            max_target_time: 1800,
+            min_difficulty: 60_000_000.into(),
+            max_difficulty: u64::MAX.into(),
+            target_time: 300,
+        });
+        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+            max_target_time: 1200,
+            min_difficulty: 70_000_000.into(),
```

Comment:
I think this is too high and will prevent mining

Reply:
Yeah, looks like this was the placeholder value from `mainnet` and not the value from esme. Updated to match the esme value.